src/EnablerExtension.php

```diff
@@ -59,9 +59,12 @@ class EnablerExtension extends Extension
     public function beforeCallActionHandler()
     {
         $config = Config::inst();
-        $action = $this->getOwner()->getAction();
+        // Routes are case-insensitive, so compare everything using lower case
+        $action = strtolower($this->getOwner()->getAction());
         $allowedActions = $config->get(Security::class, 'allowed_actions');
+        $allowedActions = array_map('strtolower', $allowedActions);
         $excludedActions = $config->get(EnablerExtension::class, 'excluded_actions');
+        $excludedActions = array_map('strtolower', $excludedActions);
```

Comment:
```suggestion
        $allowedActions = array_map('strtolower', $config->get(Security::class, 'allowed_actions'));
        $excludedActions = array_map('strtolower', $config->get(EnablerExtension::class, 'excluded_actions'));
```
No need for intermediate variables which to my eye make this harder to read

Reply:
I don't agree with this proposed style :-) method calls inside function looks kind of convoluted. Makes it harder to see where the value originally came from.

Reply:
It's exactly the same as `strtolower($this->getOwner()->getAction());` lol. Not worth arguing about though, I'll merge.